librclone/librclone.go

```diff
@@ -79,5 +82,11 @@ func RcloneRPC(method *C.char, input *C.char) (result C.struct_RcloneRPCResult)
 	return result
 }
 
+// Use this to free the string returned by RcloneRPC
```

Comment:
This comment will need to start `// RcloneFreeString may be used to free the string returned by RcloneRPC`

Perhaps command here that you can equally well use `free` from the shared C library if you have access to it.